File: cfgs/pythoncfgs/COUNT_CHARACTERS_POSITION_ENGLISH_ALPHABETS.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;fun_2(fun_2(foo_1));9;
3;statement;return foo_1;12;
4;conditional;foo_1 == fun_1(foo_2[foo_3]) - fun_2('a') or foo_4 == fun_3(foo_5[foo_6]) - fun_4('A');10;
5;statement;foo_1 += 1;11;
6;exit;;
Edges
0;2;
1;3;4;
2;6;
3;2;5;
4;2;
5;

File: cfgs/java_cfgs_foo/COUNT_CHARACTERS_POSITION_ENGLISH_ALPHABETS.java
Nodes
1;special;;;
2;statement;int result = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < fun_1();16;
5;conditional;(foo_1 == (fun_1(foo_3) - 'a')) || (foo_4 == (fun_2(foo_6) - 'A'));18;
6;statement;result++;19;
7;statement;i++;17;
8;statement;return foo_1;22;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;8;
4;6;7;
5;7;
6;4;
7;9;
8;
